CMakeLists.txt

```diff
@@ -123,16 +123,25 @@ FIND_PACKAGE (Threads REQUIRED)
 
 IF (MQTT_USE_LOG)
     MESSAGE (STATUS "Logging enabled")
-    FIND_PACKAGE (Boost 1.67.0 REQUIRED COMPONENTS system date_time log filesystem thread)
+    FIND_PACKAGE (Boost 1.67.0 REQUIRED COMPONENTS system date_time log filesystem thread program_options)
 ELSE ()
     MESSAGE (STATUS "Logging disabled")
-    FIND_PACKAGE (Boost 1.67.0 REQUIRED COMPONENTS system date_time)
+    FIND_PACKAGE (Boost 1.67.0 REQUIRED COMPONENTS system date_time program_options)
 ENDIF ()
 
 IF (MQTT_USE_TLS)
     FIND_PACKAGE (OpenSSL REQUIRED)
     IF (MQTT_USE_STATIC_OPENSSL)
-        FIND_PACKAGE (ZLIB REQUIRED)
+         IF (UNIX AND NOT APPLE)
```

Comment:
This is also related to making a static build with CI, so I should move this to separate PR

Reply:
Do you move this to other PR ? Or is it required to pass CI ?

Reply:
Is moved to other PR